extensions/blocks/google-calendar/google-calendar.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Google Calendar Block.
+ *
+ * @since 8.0.0
+ *
+ * @package Jetpack
+ */
+
+namespace Jetpack\Google_Calendar_Block;
+
+const FEATURE_NAME = 'google-calendar';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+/**
+ * Check if the block should be available on the site.
+ *
+ * @return bool
+ */
+function is_available() {
+	if (
+		defined( 'IS_WPCOM' )
+		&& IS_WPCOM
+		&& function_exists( 'has_any_blog_stickers' )
+	) {
+		if ( has_any_blog_stickers(
+			array( 'premium-plan', 'business-plan', 'ecommerce-plan' ),
+			get_current_blog_id()
+		) ) {
+			return true;
+		}
+		return false;
+	}
+
+	return true;
+}
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	if ( is_available() ) {
+		jetpack_register_block(
+			BLOCK_NAME,
+			array(
+				'render_callback'      => 'Jetpack\Google_Calendar_Block\load_assets',
+				'version_requirements' => array(
```

Comment:
Would it be worth moving this to the availability mechanism and having a new kind of nudge to get the user to upgrade or install Gutenberg? That way the block will still appear in the inserter searches.

Reply:
good idea, will take a look at how easy it will be to do this.